types/trace.go

```diff
@@ -0,0 +1,58 @@
+package types
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// TraceSession gets created alongside a pipeline
+// when tracing is enabled.
+// This will run for the configured lifespan
+// and process records during that time.
+type TraceSession struct {
+	ID         string        `json:"id" yaml:"id"`
+	PipelineID string        `json:"pipelineID" yaml:"pipelineID"`
```

Comment:
It's missing the list of plugins Plugins *string, dummy.0, syslog.0. This information will be consumed by calyptia-core to enable tracing on the specified plugins.

Reply:
So... When you create a pipeline, if you set tracing to be enabled you must also pass a list of plugins?